Added a note on Python 2 to Caffe description

evgri243 opened the pull request:
Recently, we noted that Caffe on DSVM doesn't work out of the box (even binary calls) if Python 3 is activated. The only found workaround at the moment is to switch to Python 2.7 that is installed into `root` environment with `source activate root`. Then it works just fine. Initial thread on [Microsoft Social](https://social.microsoft.com/Forums/en-US/06421319-cb30-4798-bfff-c7a90f4934a8/cannot-use-caffe-with-data-science-virtual-machine-for-linux?forum=dsvm).

azurecla commented:
Hi __@Taojunshen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-jutao). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
Added a note on Python 2 to Caffe description

Recently, we noted that Caffe on DSVM doesn't work out of the box (even binary calls) if Python 3 is activated. The only found workaround at the moment is to switch to Python 2.7 that is installed into `root` environment with `source activate root`. Then it works just fine. Initial thread on [Microsoft Social](https://social.microsoft.com/Forums/en-US/06421319-cb30-4798-bfff-c7a90f4934a8/cannot-use-caffe-with-data-science-virtual-machine-for-linux?forum=dsvm).
